iOS/IssueTracker/IssueTracker/Global/Util/Factory/IssueListViewControllerFactory.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  IssueListViewControllerFactory.swift
+//  IssueTracker
+//
+//  Created by Noah on 2023/05/25.
+//
+
+import UIKit
+
+struct IssueListViewControllerFactory: ViewControllerMakable {
+    private static let storyBoardRepresentation = StoryBoardLiteral.issue
+    
+    static func make() -> UIViewController {
```

Comment:
```Swift
protocol ViewControllerMakable {
     associatedtype ViewControllerType: UIViewController
     static func make() -> ViewControllerType
 }
```
```suggestion
struct IssueListViewControllerFactory: ViewControllerMakable {
    private static let storyBoardRepresentation = StoryBoardLiteral.issue
    
    static func make() -> IssueListViewController {
```
`ViewControllerMakable`에 associatedType을 활용해보면 어떨까요 ?

Reply:
다시보니 return type이 조금 더 명확해졌으면 좋겠네요
반영하도록 하겠습니다!!
감사합니다 :)